recipes/metocean-stats/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "metocean-stats" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0429e61fc321988da3dbe46c40aff72c931119341b99710626d5df0fbd10c09c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.9
+    - matplotlib-base >=3.1
+    - numpy >=1.17
+    - pandas
+    - scipy
+    - pytest
+    - windrose
```

Comment:
Don't get me wrong, I love that you are using one of my packages :smile: but this one is not listed in the package metatada:

https://github.com/MET-OM/metocean-stats/blob/v.0.1.1/setup.py#L18-L25

PS: The package does list `pip` as a run time dependency and I believe that is a mistake upstream.

Reply:
Thanks a lot for noticing this mistake! I just corrected:
https://github.com/MET-OM/metocean-stats/blob/v.0.1.2/setup.py 

and also updated the version (v.0.1.2) in meta.yaml 

PS: windrose is a great and useful package. Thanks for developing it!